contracts/lacchain.system/src/lacchain.system.cpp

```diff
@@ -110,7 +110,7 @@ void lacchain::newaccount( name creator, name name, const authority& owner, cons
 }
 
 void lacchain::add_new_entity(const name& entity_name,
-                              const entity_type entity_type,
+                              const node_type node_type,
```

Comment:
@rubenabix  this parameter should remain `const entity_type entity_type,` it will be required for adding new entities